images/win/toolset-2019.json

```diff
@@ -47,6 +47,16 @@
                 "12.*",
                 "14.*"
             ]
+        },
+        {
+            "name": "Boost",
+            "url" : "https://raw.githubusercontent.com/akv-platform/boost-hostedtoolcache/master/versions-manifest.json",
+            "arch": "x64",
+            "platform" : "win32",
+            "versions": [
+                "1.69.0",

```

Comment:
Version `1.71.0` has two different toolsets `msvc14.1` and `msvc14.2`. How will we choose between them?